docs/modules/ROOT/pages/how-tos/multi-instance.adoc

```diff
@@ -0,0 +1,60 @@
+= Deploy multiple instances
+
+This guide provides an example how to deploy multiple instances of this component.
+
+== Requirements
+
+
+. Prepare catalog by configuring 2 instances
++
+[source,yaml]
+----
+applications:
+  - exoscale-metrics-collector as collector-ch-gva-2 # alias collector-ch-gva-2
+  - exoscale-metrics-collector as collector-rma-0 # alias collector-rma-0
+parameters:
+  appuio_cloud_reporting:
+    namespace: 'appuio-cloud-reporting'
+    database:
+      name: 'reporting'
+      host: 'reporting-db.appuio-reporting.svc'
+      parameters: 'sslmode=disable'
+      password: 'passw0rd'
+      port: 5432
+  collector_ch_gva_2:
+    namespace: 'appuio-cloud-reporting'
+    secrets:
+      cronjob:
+        stringData:
+          EXOSCALE_API_KEY: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/exoscale-key}"
+          EXOSCALE_API_SECRET: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/exoscale-secret}"
+          K8S_SERVER_URL: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/cluster-server}"
+          K8S_TOKEN: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/cluster-token}"
+  collector_rma_0:
+    namespace: 'appuio-cloud-reporting'
+    secrets:
+      cronjob:
+        stringData:
+          EXOSCALE_API_KEY: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/exoscale-key}"
+          EXOSCALE_API_SECRET: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/exoscale-secret}"
+          K8S_SERVER_URL: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/cluster-server}"
+          K8S_TOKEN: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/cluster-token}"
+----
++
```

Comment:
I don't want to add cloudscale into the mix in the docs, though I will change the adoc format.

Reply:
We have 2 dynamic sources here, we cannot assign the name of 1 source to the alias.

Reply:
This suggestion is related to the previous one,
see here:
```
  - exoscale-metrics-collector as collector-exoscale-ch-gva-2 <1>
```
the suffix `<1>` adds a small footnote indicator and the suggestion here is describing the footnote. Instead of a comment in YAML :)

Reply:
Thanks for explanation but it was not about the footnote but the naming, let's not discuss this further, I will add the name as you suggested.